### ISSUE TITLE
Failed to forward phosphor-host-state-manager to remote log server

### ISSUE THREAD

**Issue by gkeishin** [openbmc/phosphor-logging#14]
```
# journalctl --no-pager | grep phosphor-host-state-manager
Sep 19 19:19:49 xx.xx.xx.xx phosphor-host-state-manager[1362]: Initial Host State will be Off
Sep 19 19:19:49 xx.xx.xx.xx phosphor-host-state-manager[1362]: External request to reset reboot count
Sep 19 19:21:33 xx.xx.xx.xx phosphor-host-state-manager[1362]: Host State transaction request
Sep 19 19:21:37 xx.xx.xx.xx phosphor-host-state-manager[1362]: External request to reset reboot count
Sep 19 19:21:53 xx.xx.xx.xx phosphor-host-state-manager[1362]: Received signal that host is running
Sep 19 19:21:53 xx.xx.xx.xx phosphor-host-state-manager[1362]: Change to Host State
Sep 19 19:23:18 xx.xx.xx.xx phosphor-host-state-manager[1362]: External request to reset reboot count
Sep 20 04:54:49 xx.xx.xx.xx phosphor-host-state-manager[1362]: Host State transaction request
Sep 20 04:54:56 xx.xx.xx.xx phosphor-host-state-manager[1362]: Received signal that host is off
Sep 20 04:54:56 xx.xx.xx.xx phosphor-host-state-manager[1362]: Change to Host State
Sep 20 04:59:56 xx.xx.xx.xx phosphor-host-state-manager[1362]: Host State transaction request
Sep 20 04:59:59 xx.xx.xx.xx phosphor-host-state-manager[1362]: Received signal that host is off
Sep 20 04:59:59 xx.xx.xx.xx phosphor-host-state-manager[1362]: Change to Host State
Sep 20 04:59:59 xx.xx.xx.xx phosphor-host-state-manager[1362]: External request to reset reboot count
Sep 20 05:00:13 xx.xx.xx.xx phosphor-host-state-manager[1362]: Received signal that host is running
Sep 20 05:00:13 xx.xx.xx.xx phosphor-host-state-manager[1362]: Change to Host State
Sep 20 05:01:29 xx.xx.xx.xx phosphor-host-state-manager[1362]: External request to reset reboot count
```
Not sure if its doing selectively but I don't see this forwarded to remote log server in the latest build

**Comment by geissonator** [comment]
Was anything showing up in the remote log server @gkeishin ?  Do you have that output if so?

**Comment by gkeishin** [comment]
`From BMC:`
```
root@xx.xx.xx.xx:~# journalctl --no-pager | grep phosphor-host-state-manager
Sep 20 13:21:30 xx.xx.xx.xx phosphor-host-state-manager[1357]: Host State transaction request
Sep 20 13:21:34 xx.xx.xx.xx phosphor-host-state-manager[1357]: Received signal that host is off
Sep 20 13:21:34 xx.xx.xx.xx phosphor-host-state-manager[1357]: Change to Host State
Sep 20 13:21:34 xx.xx.xx.xx phosphor-host-state-manager[1357]: External request to reset reboot count
Sep 20 13:21:48 xx.xx.xx.xx phosphor-host-state-manager[1357]: Received signal that host is running
Sep 20 13:21:48 xx.xx.xx.xx phosphor-host-state-manager[1357]: Change to Host State
Sep 20 13:23:03 xx.xx.xx.xx phosphor-host-state-manager[1357]: External request to reset reboot count
root@xx.xx.xx.xx:~#
```

`From remote server log:`
```
[root@angelhelperxxxx ~]# cat /var/log/syslog | grep xx.xx.xx.xx | grep phosphor-host-state-manager
Sep 20 13:21:30 xx.xx.xx.xx phosphor-host-state-manager[1357 Host State transaction request
Sep 20 13:21:34 xx.xx.xx.xx phosphor-host-state-manager[1357 Received signal that host is off
Sep 20 13:21:34 xx.xx.xx.xx phosphor-host-state-manager[1357 Change to Host State
Sep 20 13:21:48 xx.xx.xx.xx phosphor-host-state-manager[1357 Received signal that host is running
Sep 20 13:21:48 xx.xx.xx.xx phosphor-host-state-manager[1357 Change to Host State
[root@angelhelperxxxx ~]#
```

The missing data on remote is `External request to reset reboot count`

**Comment by dkodihal** [comment]
```2018-10-16T09:48:38+00:00 witherspoon systemd[1]: xyz.openbmc_project.State.Host.service: Main process exited, code=killed, status=9/KILL
2018-10-16T09:48:38+00:00 witherspoon systemd[1]: xyz.openbmc_project.State.Host.service: Failed with result 'signal'.
2018-10-16T09:48:39+00:00 witherspoon systemd[1]: xyz.openbmc_project.State.Host.service: Service hold-off time over, scheduling restart.
2018-10-16T09:48:39+00:00 witherspoon systemd[1]: xyz.openbmc_project.State.Host.service: Scheduled restart job, restart counter is at 1.
2018-10-16T09:48:39+00:00 witherspoon systemd[1]: Stopping UART render controller...
2018-10-16T09:48:39+00:00 witherspoon systemd[1]: Stopped UART render controller.
2018-10-16T09:48:39+00:00 witherspoon systemd[1]: Stopped Phosphor Host State Manager.
2018-10-16T09:48:39+00:00 witherspoon systemd[1]: Starting Phosphor Host State Manager...
2018-10-16T09:48:39+00:00 witherspoon phosphor-host-state-manager[1779 Initial Host State will be Off
2018-10-16T09:48:39+00:00 witherspoon phosphor-host-state-manager[1779 External request to reset reboot count
2018-10-16T09:48:39+00:00 witherspoon systemd[1]: Started Phosphor Host State Manager.
2018-10-16T09:48:39+00:00 witherspoon systemd[1]: Started UART render controller.```

**Comment by dkodihal** [comment]
This is working for me. With rsyslog setup, I just restarted phosphor-host-state-manager. I see the traces that I posted in the previous comment at the remote.

For the remote config, I used https://github.com/openbmc/openbmc/blob/master/meta-openembedded/meta-oe/recipes-extended/rsyslog/rsyslog/rsyslog.conf (uncomment lines 81, 82).

I see all of the above messages in `/var/log/syslog` and some of them in `/var/log/messages`, but that's just due to the config.
